launch/test/launch/test_executable.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Southwest Research Institute, All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# DISTRIBUTION A. Approved for public release; distribution unlimited.
+# OPSEC #4584.
+#
+# Delivered to the U.S. Government with Unlimited Rights, as defined in DFARS
+# Part 252.227-7013 or 7014 (Feb 2014).
+#
+# This notice must appear in all copies of this file and its derivatives.
```

Comment:
@tfoote would you give this preamble a look? I'm not well versed in such licensing qualifications.

Reply:
@tfoote friendly ping

Reply:
@roger-strain Does this notice have to be on these files?  It is much easier for us if this is just not there; then it looks the same as all of the rest of the licensing in our code and isn't a special snowflake.  If it absolutely has to be there for some reason, I'm going to have to check with some others about whether this is allowed in this codebase.

Reply:
It's really the whole thing.  The basic problem is that we are not lawyers, and so that anything that deviates from the standard license text is a problem.

Please let me know what you decide.  Once you've come to your decision, I'll pass it along to those more in the know on this side.  Thanks.

Reply:
Just to follow up, we're still in active discussions about how to sort this out. Appreciate your patience.